JamitFoundationExample/JamitFoundationExample.xcodeproj/project.pbxproj

```diff
@@ -1115,65 +1146,70 @@
 /* End XCConfigurationList section */
 
 /* Begin XCRemoteSwiftPackageReference section */
-		1EBCCBC425F4F0BC00079E3F /* XCRemoteSwiftPackageReference "JamitFoundation" */ = {
+		C54191312860AD9C00E29C3B /* XCRemoteSwiftPackageReference "JamitFoundation" */ = {
 			isa = XCRemoteSwiftPackageReference;
-			repositoryURL = "https://github.com/JamitLabs/JamitFoundation";
+			repositoryURL = "git@github.com:JamitLabs/JamitFoundation.git";
 			requirement = {
-				branch = develop;
+				branch = "feature/swiftui-integration";
```

Comment:
I'm sure changing this back just before merging this PR is already on your mind, but just wanted to drop a reminder to make sure.